---
Exception\UnpreparedException:
  comment: |-
    UnpreparedException is raised when a given prepared statement id does not
    exist on the server. The driver should be automatically re-preparing the
    statement in this case. Seeing this error could be considered a bug.
  methods:
    __construct:
      comment: ""
      params:
        message:
          comment: ""
          type: string
        code:
          comment: ""
          type: int
        previous:
          comment: ""
          type: \Exception
    __wakeup:
      comment: ""
      return:
        comment: ""
        type: void
    __toString:
      comment: ""
      return:
        comment: ""
        type: string
...
